packages/react-native-sdk/src/components/Call/CallControls/CallControlsButton.tsx

```diff
@@ -8,6 +8,7 @@ import {
 } from 'react-native';
 import { useTheme } from '../../../contexts/ThemeContext';
 import { Theme } from '../../../theme/theme';
+import { colorPalette } from '../../..';
```

Comment:
This should be an illegal import as it is source of theme bugs :)

Reply:
we should never import color palette inside our SDK components

But we should export colorPalette to integrators as they may want to use the default theme colors for their own components outside of `useTheme`